src/main/java/com/iridium/iridiumskyblock/DataConverter.java

```diff
@@ -0,0 +1,107 @@
+                        IridiumSkyblock.getInstance().getLogger().info("Copying " + file.getName() + " to " + target.toFile().getAbsolutePath() + "...");
+                        Files.copy(source, target);
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                });
+
+        Arrays.stream(Objects.requireNonNull(new File(IridiumSkyblock.getInstance().getDataFolder().getAbsolutePath() + File.separator + "schematics" + File.separator)
+                        .listFiles((file, fileName) -> fileName.endsWith(".schem") || fileName.endsWith(".schematic"))))
+                .forEach(file -> {
+                    try {
+                        Path source = Paths.get(file.getAbsolutePath());
+                        Path target = Paths.get(file.getParentFile().getParentFile().getAbsolutePath() + File.separator + "old_data" + File.separator + "schematics" + File.separator + file.getName());
+                        IridiumSkyblock.getInstance().getLogger().info("Copying " + file.getName() + " to " + target.toFile().getAbsolutePath() + "...");
+                        Files.copy(source, target);
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                });
+    }
+
+
+    public static void updateDatabaseData(int oldVersion, int newVersion, ConnectionSource connectionSource, SQL.Driver driver) {
+        IridiumSkyblock.getInstance().getLogger().info("Updating database from version " + oldVersion + " to " + newVersion);
+
+        try {
+            DatabaseConnection connection = connectionSource.getReadWriteConnection(null);
+            for (int version = oldVersion + 1; version <= newVersion; version++) {
+                InputStream inputStream = findPatchInputStream(version, driver);
+                if (inputStream == null) continue;
+
+                for (String statement : CharStreams.toString(new InputStreamReader(inputStream)).split("\n")) {
+                    connection.executeStatement(statement, DatabaseConnection.DEFAULT_RESULT_FLAGS);
+                }
+            }
+
+            connectionSource.releaseConnection(connection);
+            IridiumSkyblock.getInstance().getLogger().info("Update successful!");
+        } catch (SQLException | IOException exception) {
+            exception.printStackTrace();
+        }
+    }
+
+    private static InputStream findPatchInputStream(int version, SQL.Driver driver) {
+        InputStream inputStream = IridiumSkyblock.getInstance().getResource("patch_" + version + ".sql");
+        if (inputStream == null) {
+            inputStream = IridiumSkyblock.getInstance().getResource("patch_" + version + "_" + driver.name().toLowerCase() + ".sql");
+        }
+
+        return inputStream;
+    }
+
+    public void convert() {
+
+    }
```

Comment:
This doesnt look like it would work? you have some empty methods, and there are no patch_.sql files